src/autoscaler/api/brokerserver/broker_handler.go

```diff
@@ -452,6 +452,10 @@ func (h *BrokerHandler) UpdateServiceInstance(w http.ResponseWriter, r *http.Req
 	}
 }
 
+func (h *BrokerHandler) GetBrokerCatalogPlanId(instanceId string) (string, error) {
+	return h.cfClient.GetServicePlan(instanceId)
```

Comment:
This is an interim step where I think moving the `joins` from the cf resources is better done here.